src/hotspot/share/gc/shenandoah/heuristics/shenandoahOldHeuristics.cpp

```diff
@@ -88,7 +87,8 @@ bool ShenandoahOldHeuristics::prime_collection_set(ShenandoahCollectionSet* coll
   // of memory that can still be evacuated.  We address this by reducing the evacuation budget by the amount
   // of live memory in that region and by the amount of unallocated memory in that region if the evacuation
   // budget is constrained by availability of free memory.
-  size_t old_evacuation_budget = (size_t) ((double) heap->get_old_evac_reserve() / ShenandoahOldEvacWaste);
+  size_t old_evacuation_reserve = heap->old_generation()->get_evacuation_reserve();
+  size_t old_evacuation_budget = (size_t) ((double) old_evacuation_reserve / ShenandoahOldEvacWaste);
```

Comment:
const

Reply:
There may be others too that could may be be const'ed, if you could check please (I didn't check for any others). Thanks!